src/components/app/LoginForm.js

```diff
@@ -59,6 +60,19 @@ class LoginForm extends Component {
 
     }
 
+    checkIfAlreadyLogged(err){
+        const {dispatch} = this.props;
+
+        return dispatch(localLoginRequest())
+            .then(response => {
+                if (response.data){
+                    return location.href = '/';
```

Comment:
First, if you have that error, it means that something is did wrong way. You should probably cancel some promise.
2nd, i see that i should investigate more that case, because im not sure about the solution